File: 204-count-primes/count-primes.go
func countPrimes(n int) int {
    var counter int
    primes := make([]bool, n + 2, n+ 2)
    primes[0] , primes[1] = true, true
    for i:=2; i < n; i++ {
        if (!primes[i]) {
            counter ++
            for j := i + i; j < n; j += i{
                primes[j] = true
            }
        }
    }
    return counter

}

File: 204-count-primes/count-primes.py
class Solution:
    def countPrimes(self, n: int) -> int:
        primes = [True] * (n)
        counter = 0
        for i in range(2, n):
            if primes[i]:
                counter += 1
                for j in range(i + i, n, i):
                    primes[j] = False
        return counter